<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>725</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Enchain</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QGraphicsView" name="graphicsView">
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="focusPolicy">
       <enum>Qt::WheelFocus</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QProgressBar" name="progressBar">
      <property name="value">
       <number>24</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>25</height>
    </rect>
   </property>
   <property name="nativeMenuBar">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionOnline_Help"/>
    <addaction name="actionAuthor_Info"/>
    <addaction name="actionCheck_Update"/>
    <addaction name="actionAbout_Enchain"/>
    <addaction name="actionAbout_Qt"/>
   </widget>
   <widget class="QMenu" name="menuTool">
    <property name="title">
     <string>Labelling</string>
    </property>
    <addaction name="actionShow_guidance"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionCreateVOCFolder"/>
    <addaction name="actionCutImage"/>
   </widget>
   <widget class="QMenu" name="menuManagement">
    <property name="title">
     <string>Management</string>
    </property>
    <addaction name="actionDatasetInput"/>
   </widget>
   <widget class="QMenu" name="menuPreprocess">
    <property name="title">
     <string>Preprocess</string>
    </property>
    <widget class="QMenu" name="menuImages">
     <property name="title">
      <string>Images</string>
     </property>
     <addaction name="actionOpenImageFolder"/>
     <addaction name="actionOpenImage"/>
     <addaction name="actionSaveImage"/>
    </widget>
    <widget class="QMenu" name="menuVideo">
     <property name="title">
      <string>Video</string>
     </property>
     <addaction name="actionOpenVideo"/>
     <addaction name="actionOpenVideoFolder"/>
     <addaction name="separator"/>
     <addaction name="actionSaveSliceTo"/>
    </widget>
    <widget class="QMenu" name="menuDataCleansing">
     <property name="title">
      <string>DataCleansing</string>
     </property>
     <addaction name="actionSelectSource"/>
     <addaction name="actionSelectDestination"/>
     <addaction name="separator"/>
    </widget>
    <addaction name="menuVideo"/>
    <addaction name="menuImages"/>
    <addaction name="separator"/>
    <addaction name="menuDataCleansing"/>
    <addaction name="separator"/>
    <addaction name="actionAssignment"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuPreprocess"/>
   <addaction name="menuTool"/>
   <addaction name="menuManagement"/>
   <addaction name="menuTools"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="dockWidgetRight">
   <property name="accessibleName">
    <string/>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContentsRight">
    <property name="accessibleName">
     <string/>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QListView" name="listView"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidgetLeft">
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContentsLeft">
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QTreeWidget" name="treeWidget">
       <column>
        <property name="text">
         <string notr="true">1</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionAuthor_Info">
   <property name="text">
    <string>Connect Author</string>
   </property>
  </action>
  <action name="actionOnline_Help">
   <property name="text">
    <string>Online Help</string>
   </property>
  </action>
  <action name="actionCheck_Update">
   <property name="text">
    <string>Check Update</string>
   </property>
  </action>
  <action name="actionOpenImageFolder">
   <property name="text">
    <string>OpenImageFolder</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionOpenVideo">
   <property name="text">
    <string>OpenVideo</string>
   </property>
  </action>
  <action name="actionOpenImage">
   <property name="text">
    <string>OpenImage</string>
   </property>
  </action>
  <action name="actionSaveImage">
   <property name="text">
    <string>SaveImage</string>
   </property>
  </action>
  <action name="actionCutImage">
   <property name="text">
    <string>CutImage</string>
   </property>
  </action>
  <action name="actionDatasetInput">
   <property name="text">
    <string>DatasetManage</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionShow_guidance">
   <property name="text">
    <string>Guidance Wiki</string>
   </property>
  </action>
  <action name="actionSaveSliceTo">
   <property name="text">
    <string>SaveSliceTo</string>
   </property>
  </action>
  <action name="actionSelectSource">
   <property name="text">
    <string>SelectSource</string>
   </property>
  </action>
  <action name="actionSelectDestination">
   <property name="text">
    <string>SelectDestination</string>
   </property>
  </action>
  <action name="actionAssignment">
   <property name="text">
    <string>Assignment</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionAbout_Enchain">
   <property name="text">
    <string>About Enchain</string>
   </property>
  </action>
  <action name="actionCreateVOCFolder">
   <property name="text">
    <string>CreateVOCFolder</string>
   </property>
  </action>
  <action name="actionOpenVideoFolder">
   <property name="text">
    <string>OpenVideoFolder</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>treeWidget</tabstop>
  <tabstop>listView</tabstop>
  <tabstop>graphicsView</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
